### PR TITLE
Changed all instances of stamenmap to stadiamap with maptype changes.

### PR DESCRIPTION
**Pull request by jackVanish**
Changed all instances of stamenmaps to stadiamaps, along with maptypes.

### CODE REVIEW

File: code/day1/day1_MIGRAMAR_Node.R [hunk 1]
```diff
@@ -246,12 +246,12 @@ library(ggmap)
...
 
 
-base <- get_stamenmap(
+base <- get_stadiamap(
   bbox = c(left = min(gmr_deploy$DEPLOY_LONG),
            bottom = min(gmr_deploy$DEPLOY_LAT),
            right = max(gmr_deploy$DEPLOY_LONG),
            top = max(gmr_deploy$DEPLOY_LAT)),
-  maptype = "terrain",
+  maptype = "stamen_terrain",
```
Comment (CaitlinBate) [inline code/day1/day1_MIGRAMAR_Node.R:254]: should this be stamen_terrain_background?

**Comment by CaitlinBate** [comment]
setup.R needs to be edited to include the stadiamap API key

additionally, the OTN version of day1 code is OK and doesn't need updating to stadiamap? it wasn't edited here

File: code/day1/day1_MIGRAMAR_Node.R [hunk 2]
```diff
@@ -397,12 +397,12 @@ gmr_matched_18_19_no_release <- gmr_matched_18_19  %>%
...
 
 ## Detection Map - Static ----
-base <- get_stamenmap(
+base <- get_stadiamap(
   bbox = c(left = min(gmr_matched_18_19_no_release$longitude),
            bottom = min(gmr_matched_18_19_no_release$latitude), 
            right = max(gmr_matched_18_19_no_release$longitude), 
            top = max(gmr_matched_18_19_no_release$latitude)),
-  maptype = "terrain", 
+  maptype = "stamen_terrain",
```
[the remark this answers is not in the document]
Comment (jackVanish) [reply code/day1/day1_MIGRAMAR_Node.R:405]: In both cases I updated the style to the stadiamaps equivalent; terrain and terrain-background are both valid stamenmap styles and so stamen_terrain and stamen_terrain_background are both valid stadiamaps backgrounds. I'm using what was already there; we can change it to stamen_terrain_background if we like but it isn't a functional problem.

**Comment by jackVanish** [comment]
API key added. Also, yeah, the day1 OTN code doesn't need to be updated. It was pulled over from the CANSSI workshop, so it already had stadiamaps in it.

**Pull request merged**